plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java

```diff
@@ -67,16 +87,10 @@
 import org.apache.maven.shared.filtering.MavenFileFilter;
 import org.apache.maven.shared.filtering.MavenFilteringException;
 
-import javax.validation.ConstraintViolationException;
-import java.io.File;
-import java.io.FileFilter;
-import java.io.IOException;
-import java.util.*;
-
-import static io.fabric8.maven.core.util.Constants.*;
+import static io.fabric8.maven.core.util.Constants.RESOURCE_APP_CATALOG_ANNOTATION;
 import static io.fabric8.maven.core.util.KubernetesResourceUtil.isAppCatalogResource;
+import static io.fabric8.maven.core.util.KubernetesResourceUtil.loadResources;
 import static io.fabric8.maven.plugin.mojo.build.ApplyMojo.DEFAULT_OPENSHIFT_MANIFEST;
-import static io.fabric8.maven.plugin.mojo.build.ApplyMojo.loadResources;
```

Comment:
Yeah, I'm not even a fan of classes for constants and I've seen too many projects with 100+ constants in a `utils.Constants` file... The value of that constant will go eventually into a parameter of the `AbstractFabric8Mojo`, but we need to break some inheritance from the `d-m-p` project mojos before ;)

I think it's natural to define a set of utility classes and then start putting stuff into them, eventually filling them with unrelated things. But you know, when you develop a new thing, you never create a utility class for holding a single method. When an utility class grows (`KubernetesResourceUtil` is one of them..) you start splitting it in multiple utility classes or you extract a service from it. The good thing is that refactoring utility classes is always much easier than refactoring a hierarchy of dependent mojos :)